docs/jobs.md

```diff
@@ -230,26 +237,26 @@ Accept: application/vnd.api+json
 
 ```json
 {
-    "data": {
-        "type": "io.cozy.jobs",
-        "id": "123123",
-        "attributes": {
-            "domain": "me.cozy.tools",
-            "worker": "sendmail",
-            "options": {
-                "priority": 3,
-                "timeout": 60,
-                "max_exec_count": 3
-            },
-            "state": "running",
-            "queued_at": "2016-09-19T12:35:08Z",
-            "started_at": "2016-09-19T12:35:08Z",
-            "error": ""
-        },
-        "links": {
-            "self": "/jobs/123123"
-        }
+  "data": {
+    "type": "io.cozy.jobs",
+    "id": "123123",
+    "attributes": {
+      "domain": "me.cozy.tools",
+      "worker": "sendmail",
+      "options": {
+        "priority": 3,
+        "timeout": 60,
+        "max_exec_count": 3
+      },
+      "state": "running",
+      "queued_at": "2016-09-19T12:35:08Z",
+      "started_at": "2016-09-19T12:35:08Z",
+      "error": ""
+    },
+    "links": {
+      "self": "/jobs/123123"
     }
+  }
```

Comment:
Why are arguments not returned in the response ? Is this normal ?

Reply:
They are returned in the response, but encoded in base64 (for historical technical reasons). So, I prefer to omit them in the documentation to avoid confusion.